db-scripts/src/main/resources/cgds.sql

```diff
@@ -266,18 +268,33 @@ CREATE TABLE `uniprot_id_mapping` (
 );
 
 -- --------------------------------------------------------
+DROP TABLE IF EXISTS `reference_genome`;
+CREATE TABLE `reference_genome` (
+    `reference_genome_id` int(11) NOT NULL AUTO_INCREMENT,
+    `species` varchar(64) DEFAULT NULL,
+    `name` varchar(64) DEFAULT NULL,
+    `build_name` varchar(64) DEFAULT NULL,
+    `genome_size` bigint(20) DEFAULT NULL,
+    `URL` varchar(256) DEFAULT NULL,
+    `release_date` datetime DEFAULT NULL,
+    PRIMARY KEY (`reference_genome_id`)
+);
```

Comment:
Did you want to create an index on build_name here?

Reply:
sorry, I was away. 
it is a relative small table, tracking a couple of reference genomes . It might not help much.